app/controllers/admin/case_replies_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Admin
+  class CaseRepliesController < AdminController
+    def index
+      @open_cases = Case.where(status: Case::OPEN)
+      @assigned_cases = Case.where(assignee: current_user, status: Case::IN_PROGRESS)
+
+      @replies = CaseReply.all
+    end
+
+    def create
+      CaseReply.create(reply_params)
+      redirect_to admin_case_replies_path, flash: { notice: "Your saved reply was created successfully."}
+    end
+
+    def edit
+      @reply = CaseReply.find(params[:id])
+      @open_cases = Case.where(status: Case::OPEN)
+      @assigned_cases = Case.where(assignee: current_user, status: Case::IN_PROGRESS)
+    end
+
+    def update
+      CaseReply.update(reply_params)
```

Comment:
This seems to update every CaseReply (which happened for me)

Reply:
Whoops, forgot a find 
https://github.com/brave-intl/publishers/pull/2054/commits/1bf5f4c9616e662e38bb12bf8a677c5b761abfd4